guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/multiply-two-decimals-with-javascript/index.md

```diff
@@ -8,3 +8,34 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+JavaScript uses the * symbol for multiplication. Multiplying decimals is the same as multiplying whole numbers except you are working with decimal numbers now so you must include the decimal point and the number that follows the decimal in your equation.
+
+For example, if you were to multiply 3 by 5 then you could simply type:
+```Javascript
+var product = 3 * 5; // product is 15.
+``` 
+But if you want to multiply 3.0 and 5.0 you would type:
+```Javascript
+var product = 3.0 * 5.0; // product is 15. 
+
+// Note: JavaScript automatically removes the decimal places because 3.0*5.0=15.0 which is a whole number.
```

Comment:
I think that these examples are making it a bit confusing. Please try to fix them!